src/Storage/Storage.Management/help/Get-AzDataLakeGen2DeletedItem.md

```diff
@@ -0,0 +1,181 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Storage.dll-Help.xml
+Module Name: Az.Storage
+online version: https://docs.microsoft.com/powershell/module/az.storage/get-azdatalakegen2deleteditem
+schema: 2.0.0
+---
+
+# Get-AzDataLakeGen2DeletedItem
+
+## SYNOPSIS
+List all deleted files or directories from a directory or filesystem root.
+
+## SYNTAX
+
+```
+Get-AzDataLakeGen2DeletedItem [-FileSystem] <String> [[-Path] <String>] [-MaxCount <Int32>]
+ [-ContinuationToken <String>] [-AsJob] [-Context <IStorageContext>] [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-AzDataLakeGen2DeletedItem** cmdlet lists all deleted files or directories from a directory or filesystem in an Azure storage account.
+This cmdlet only works if Hierarchical Namespace is enabled for the Storage account. This kind of account can be created by run "New-AzStorageAccount" cmdlet with "-EnableHierarchicalNamespace $true".
+
+## EXAMPLES
+
+### Example 1: List all deleted files or directories from a Filesystem
+```
+PS C:\> Get-AzDataLakeGen2DeletedItem -FileSystem "filesystem1" 
+
+
+   FileSystem Name: filesystem1
+
+Path                 DeletionId           DeletedOn            RemainingRetentionDays
+----                 ----------           ---------            ----------------------
+dir0/dir1/file1      132658816156507617   2021-05-19 07:06:55Z 3                     
+dir0/dir2            132658834541610122   2021-05-19 07:37:34Z 3                    
+dir0/dir2/file3      132658834534174806   2021-05-19 07:37:33Z 3
+```
```

Comment:
Please change the same to following format, or the Static analysis might fail:

````markdown
```powershell
$localUser = Get-AzStorageLocalUser ..... (note: "PS C:|>" needs to be deleted)
```

```output
......
```
````

Reply:
same as below examples

Reply:
Done